test/integration/BUILD

```diff
@@ -1682,7 +1682,9 @@ envoy_cc_test(
         "//source/common/event:dispatcher_lib",
         "//source/common/network:connection_lib",
         "//source/common/network:utility_lib",
+        "//source/extensions/bootstrap/internal_listener:config",
```

Comment:
I think this test will fail if internal listener or userspace socket extensions are disabled. I think this test needs to be moved into an extension specific directory, such that it is automatically skipped if extension is disabled. This will also avoid the visibility change you had to make for extension.

Reply:
I imagine this test binary built the new extensions, which is independent from the disabled list in //source/exe:envoy-static

Reply:
OK, i see the integration test living in the per extension path. Let me migrate this test there

Reply:
done in https://github.com/envoyproxy/envoy/pull/19467/commits/fa940135e1ddd4ef63939b034cb4b0b41927c52a

Reply:
@yanavlasov  Moved this test in the extension directory and simulated a import without this extension. Is it good to go? Thanks